### DISCUSSION TITLE
Filter or option to hide media that's already been requested

### DISCUSSION THREAD

**Discussion by ejsoler** [seerr-team/seerr#486] [Ideas]
Hi, I think a great feature to add would be the ability to be able to hide or filter off what has already been requested not only what's already available. Sometimes it takes a lot of time to find new things to add when browsing because the things that have already been requested are there mixed in.

**Comment by RubyEyedReaper** [comment]
and already owned!

**Comment by yourfavoriteweapon** [comment]
https://github.com/sct/overseerr/issues/3681 vote here

**Reply by traiannicolau** [reply]
Would love to see this.

**Reply by Tmatz27** [reply]
Unfortunately it was archived by the owner and it's stale I think we need to open a new one. Would love a button on the discover page to filter out requested and/or available content. I know it's available in the settings but it's not intuitive or easy to use

**Reply by fallenbagel** [reply]
> Unfortunately it was archived by the owner and it's stale I think we need to open a new one. 

Because overseerr is deprecated. Everything should be here now.

**Reply by Tmatz27** [reply]
> > Unfortunately it was archived by the owner and it's stale I think we need to open a new one.
> 
> Because overseerr is deprecated. Everything should be here now.

I totally didnt realize that took me away from seerr and redirected me to overseerr. I appreciate the correction and all the work you do!

**Reply by jpbriggs** [reply]
I was just drafting a request for this, when search showed me this one. But so is this actually archived? Is there no active request out there for this?

**Reply by fallenbagel** [reply]
> I was just drafting a request for this, when search showed me this one. But so is this actually archived? Is there no active request out there for this?

Overseerr is deprecated. Everything should be directed to this

**Comment by JoshuaAJones74** [comment]
I went so far as to "Create empty movie folders" when requests are entered but still cannot get Seerr to react to them being there.
If it sees an empty folder, Seerr should see that the movie has been requested.
When the folder is filled, it should report that the file is available.

Am I thinking correctly on this?

**Reply by fallenbagel** [reply]
No. Seerr doesnt care about your folders. You need to enable scan in settings > services > radarr/sonarr. Then when you add any item and have them monitored, when the sonarr/radarr scan runs, it will mark as requested. **seerr does not interact with files in any way**.

**Reply by JoshuaAJones74** [reply]
I do have scan selected on each service BUT they don't seem to work.
Ex: Project Hail Mary has been in Radarr for about a year and it still shows on the Discover pages.

What has been starting to work is using Trailarr.  It will place a trailer into the created folders and Seerr removes it from Discovery.  It doesn't add it to Recently Added until the actual movie arrives. It does say Available when there is only a trailer though.

**Comment by duhnl** [comment]
<img width="712" height="503" alt="HIDE" src="https://github.com/user-attachments/assets/f0c55647-8169-4f20-8501-7279cdd8fc28" />

I solved for my private server.

**Comment by 0xSysR3ll** [comment]
It's being worked on, see: https://github.com/seerr-team/seerr/pull/1855
